mongo-dao/src/main/java/com/kovan/mapper/NewsMapper.java

```diff
@@ -4,8 +4,6 @@
 import com.kovan.entity.NewsEntity;
 import io.micrometer.common.util.StringUtils;
 import org.springframework.stereotype.Component;
-
-import java.time.LocalDateTime;
 import java.util.Collections;
 import java.util.List;
 import java.util.Objects;
```

Comment:
In this class you have used StringUtils can you please make it as static import.

Reply:
made import as static.